includes/PerformanceFeatureHooks.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace NewfoldLabs\WP\Module\Performance;
+
+
+use NewfoldLabs\WP\ModuleLoader\Container;
+use NewfoldLabs\WP\Module\Performance\CacheTypes\Browser;
+use NewfoldLabs\WP\Module\Performance\CacheTypes\File;
+use NewfoldLabs\WP\Module\Performance\CacheTypes\Skip404;
+use NewfoldLabs\WP\Module\Performance\ResponseHeaderManager;
+
+use function NewfoldLabs\WP\Module\Performance\getCacheLevel;
+use function NewfoldLabs\WP\ModuleLoader\container as getContainer;
+use function NewfoldLabs\WP\Context\getContext;
+use function NewfoldLabs\WP\Module\Features\disable as disableFeature;
+use function NewfoldLabs\WP\Module\Features\isEnabled;
+
+/**
+ * This class adds performance feature hooks.
+ **/
+class PerformanceFeatureHooks {
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		if ( function_exists( 'add_action' ) ) {
+			add_action( 'newfold_container_set', array( $this, 'pluginHooks') );
+		}
+	}
+
+	/**
+	 * Hooks for plugin activation/deactivation
+	 */
+	public function pluginHooks( Container $container ) {
+		register_activation_hook( $container->plugin()->file, array( $this, 'onActivation' ) );
+		register_deactivation_hook( $container->plugin()->file, array( $this, 'onDeactivation' ) );
+	}
+
+	/**
+	 * Activation hook to perform when plugin is activated or feature is enabled
+	 */
+	public function onActivation() {
```

Comment:
@circlecube Looks good, but let's also perform the `onActivation` when the module is turned on and the `onDeactivation` when the module is turned off.